helpers.json

```diff
@@ -129,8 +228,28 @@
     "name": "canKillStandingGreenSpacePirates",
     "requires": [
       {"or": [
-        "Missile",
-        "Super",
+        {"and": [
+          "Missile",
+          {"cost":[
+            {"ammo": [
+              {
+                "type": "Missile",
+                "count": 1
```

Comment:
We shouldn't hard-code values that can be inferred from the enemy data. I think this applies to all other enemy-based helpers too.

Reply:
Yeah I've been unsure what to do with these.